build.gradle

```diff
@@ -34,6 +34,7 @@ allprojects {
         commonsIoVersion = '2.5'
         commonsLangVersion = '3.12.0'
         gltfVersion = '2.1.0'
+        args4jVersion = '2.32'
```

Comment:
Why not use the latest 2.33?

Reply:
No particular reason, when I searched maven earlier only 2.32 was coming up but I updated to 2.33 and its working.